repo/meta/index.json

```diff
@@ -29,6 +72,22 @@
       }
     },
     {
```

Comment:
is anyone updating the install docs so that people trying to install
kubernetes aren't stymied by the fact that multiverse isn't part of the
default dcos cli config (or is that changing too)?

On Tue, Sep 22, 2015 at 5:24 PM, Ken Sipe <notifications@github.com> wrote:

> In repo/meta/index.json
> <https://github.com/mesosphere/multiverse/pull/5#discussion_r40146435>:
>
> > @@ -29,6 +72,22 @@
> >        }
> >      },
> >      {
>
> @jdef <https://github.com/jdef> how the heck did you catch that so
> quickly? @elingg <https://github.com/elingg> catch it as well. on it.
> This is PR that is looking for agreement and approval still. So even after
> it is technically correct... there may be some delay (I hope not much). I
> expect it to happen this week.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/5/files#r40146435>.
>